tests/core/test_evaluation.py

```diff
@@ -495,3 +495,78 @@ def test_log_evaluation_table(caplog, skip_field, skip_value):
         assert f"Classification report: \n{kwargs['report']}" in caplog.text
     else:
         assert "Classification report:" not in caplog.text
+
+
+@pytest.mark.parametrize(
+    "test_file",
+    ["data/test_yaml_stories/test_prediction_with_correct_intent_wrong_entity.yml",],
```

Comment:
why paramatrize if there is only one option?

Reply:
maybe these three tests can be combined?

Reply:
> maybe these three tests can be combined?

True! I forgot to change it because I had everything in one test before.

> maybe these three tests can be combined?

I had it like this before with an extra variable parametrized and then inside the test function use this variable in `if` statements. But then I remembered a comment from Tobias [here](https://github.com/RasaHQ/rasa/pull/7931#discussion_r615957054) where he alternatively suggests to split in different tests because the if statements makes it hard to read. But maybe it is because in that case I had a whole domain added in every if statement. 

I will put them all in one and maybe in this case it is not that hard to read.

Reply:
I put the 3 tests into the same function and it actually goes faster because before each test was taking ~30seconds to finish so in total 90seconds all 3 and now that I have them in the same function it takes ~31seconds to complete. So very good call Joe!!! 👌🏾This happens because it does the formbot training once right?